libafl/src/feedbacks/nautilus.rs

```diff
@@ -82,10 +82,7 @@ impl<'a, S> Named for NautilusFeedback<'a, S> {
 
 impl<'a, S> Feedback<S> for NautilusFeedback<'a, S>
 where
-    S: HasMetadata
```

Comment:
Same pattern is in here..?

Would this be a problem?

Reply:
This one is fine -- anywhere you see `<Input = ...>` automatically gets flattened out to UsesInput's `Input`.